src/console_implementation/game_loop_functions.cpp

```diff
@@ -113,4 +137,34 @@ void update(WINDOW* operating_window, std::shared_ptr<drawable_player> main_play
     main_player -> draw(main_player -> get_current_position());
     wmove(operating_window, 0, 0);
     wrefresh(operating_window);
+}
+
+std::string map_converter(std::string p_file_name, unsigned int* map_height, unsigned int* map_width) //Converting map layout from .txt file to single string
+{
+    std::ifstream file(p_file_name);
+    if(!file)
+    {
+        std::cout << "Error reading map file";
+    }
```

Comment:
After this guard clause, function continues, but I guess it shouldn't.
```suggestion
    if(!file)
    {
        std::cout << "Error reading map file";
        throw std::exception();
    }
```